InfiniteScrollView.js

```diff
@@ -116,18 +117,23 @@ class InfiniteScrollView extends React.Component {
   }
 
   _distanceFromEnd(event): number {
-    let {
+    const {
```

Comment:
Just because they happen not to be modified doesn't mean that they're semantically constant. Also we prefer `let` for cosmetic reasons.